index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Ja'fer's Portfolio</title>
+</head>
+<body>
+  <nav class="header">
+    
+    <p class="welcome"> <strong>Welcome</strong></p>
+    <div class="vector_container">
+      <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3" />
+      </svg>
+    </a>
+    <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3"  />
+      </svg>
+    </a>
+    <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3"  />
+      </svg>
+    </a>
+    </div>
+  </nav>
+  <main class="mainSection">
+    <article class="pageSection">
+      <h1 class="stText">
+        <b>Hey there. I’m Mariane</b>
+      </h1><h1 class="ndText">
+        <b>I’m a software developer</b>
+      </h1>
+    </article>
+    <article class="pageSection">
+      <p class="text">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </article>
+    <section class="pageSection socialMedia">
+      <a href="#" class="fa fa-twitter"></a>
+      <a href="#" class="fa fa-github"></a>
+      <a href="#" class="fa fa-linkedin"></a>
+      <a href="#" class="fa fa-medium"></a>
+      <a href="#" class="fa fa-angellist"></a>
+    </section>
```

Comment:
- [ ] Your Icons should be list using `<ul><li></li></ul>`, This will make you html codes more clean and organized. 
- [ ]  Replace section with 'ul' and make 'section tag' to cover the whole headline part after the toolbar.